open-budget-il/main.py

```diff
@@ -677,22 +680,23 @@ def get_query(self):
         index = search.Index(name="OpenBudget")
         results = []
         try:
-            searchQueryStringList = [queryString]
```

Comment:
you're right.

I'll merge soon.

Adam

On Thu, May 21, 2015 at 11:43 AM Yonatan Naor <notifications@github.com>
wrote:

> In open-budget-il/main.py
> <https://github.com/OpenBudget/open-budget-api/pull/7#discussion_r30782994>
> :
>
> > @@ -677,22 +680,23 @@ def get_query(self):
> >          index = search.Index(name="OpenBudget")
> >          results = []
> >          try:
> > -            searchQueryStringList = [queryString]
>
> Actually the resulting query will be:
> מדינה AND ((type=Entity) OR (type=BudgetLine AND year=2014))
>
> inside get_search_query if the handler contains FTS_FIELDS it will always
> use at least the type filter.
> Then it will report how many parameters were used in the query, if at
> least 1 handler used anything more than the type filter we will use the
> long form.
>
> I should probably move the decision logic inside the handler and return a
> boolean instead of a number.
> but the functionality is ok as it is
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenBudget/open-budget-api/pull/7/files#r30782994>.
>